ts.js

```diff
@@ -137,63 +130,7 @@
             "allowConciseArrowFunctionExpressionsStartingWithVoid": false
          }
       ],
-      '@typescript-eslint/no-inferrable-types': ['error', {'ignoreParameters': true}],
-      'arrow-body-style': [
-         'error', 
-         'as-needed'
-      ],
-      'complexity': 'off',
-      'constructor-super': 'error',
-      'curly': 'error',
-      'guard-for-in': 'error',
-      'id-blacklist': [
-         'error',
-         'any',
-         'Number',
-         'number',
-         'String',
-         'string',
-         'Boolean',
-         'boolean',
-         'Undefined',
-         'undefined'
-      ],
-      'id-match': 'error',
-      'max-classes-per-file': [
-         'error',
-         1
-      ],
-      'max-len': 'off',
-      'no-caller': 'error',
-      'no-cond-assign': 'error',
-      'no-debugger': 'error',
-      'no-empty': 'error',
-      'no-fallthrough': 'off',
-      'no-invalid-this': 'off',
-      'no-underscore-dangle': 'error',
-      'no-unsafe-finally': 'error',
-      'no-unused-labels': 'error',
-      'prefer-arrow/prefer-arrow-functions': 'error',
```

Comment:
this is sort of the section you reconciled?  none of these were really 'dropped'?

Reply:
yea. I shoulda made my commits better, but yea none of these are effectively dropped, assuming I didn't make any oopsies. These were all redundant with something being inherited in one way or another.